services/director/src/simcore_service_director/config.py

```diff
@@ -38,6 +38,7 @@
 }
 
 # some services need to know the published host to be functional (paraview)
+# TODO: please review if needed

```

Comment:
yes I think this was needed for paraview. maybe it is not anymore since it is reverse-proxied.  not sure.. will have to review this.